octovis/package.xml

```diff
@@ -17,12 +17,7 @@
 
   <exec_depend>ament_cmake</exec_depend>
```

Comment:
Can we push down this?

Reply:
Is not needed, removed here https://github.com/anasarrak/octomap/commit/6e7b57f55d93ad560cd1f036cc4de7f2b53f2720, with buildtool_depend sould be enough